lib/onfido.rb

```diff
@@ -10,7 +10,6 @@
 require 'onfido/errors/request_error'
 require 'onfido/errors/server_error'
 require 'onfido/errors/connection_error'
-require 'onfido/null_logger'
```

Comment:
Why is this change in the MR?

Reply:
If you look at the commit prior to removing this, it seems like this was causing it to fail: do you know what `null_logger` is actually doing?

Reply:
Hmm yeah that file doesn't exist - not sure how this was working before. 🤔

Reply:
Yeah it's strange, I'd have expected CI to be failing on master with the file missing. I think it's okay to keep the removal in as it doesn't exist.